src/main/kotlin/Network.kt

```diff
@@ -0,0 +1,29 @@
+class Network() {
```

Comment:
Like this: (maybe not quite right, but I think you'll get my point.)
fun readCsvInput3():MutableList<Int> {
+    val settings = CsvParserSettings()
+    settings.isHeaderExtractionEnabled = true
+    val parser = CsvParser(settings)
+    settings.format.setLineSeparator("\n")
+    val csvParser = CsvParser(settings)
+    val reader = FileAccess().getReader("/input.csv")
+    val allRows = csvParser.parseAllRecords(reader)
+
+    val network = Network()
+    var carsThatAreAllowedToDrive = .inputCarsThatAreAllowedToDriveCSV
+    for (record in allRows){
+        carsThatAreAllowedToDrive.add(record.values.get(1).toInt())
+    }
+    return carsThatAreAllowedToDrive